### PR TITLE
add 409 error support to frontend

### PR DESCRIPTION
**Pull request by heyitsmaimai**

### CODE REVIEW

File: src/events/order/cd-event-sessions.vue [hunk 1]
```diff
@@ -27,6 +27,7 @@
     </div>
     <div class="cd-event-sessions__next-block" >
       <p v-show="errors.has('submitApplications:required')" class="cd-event-sessions__next-ticket-select-error text-danger"> {{ $t('There are some errors with your booking. Please scroll up and follow the tips to finish this booking.') }}</p>
+      <p v-show="errors.has('409')" class="text-danger">{{ $t('There are some errors with your booking: not enough ticket capacity') }}</p>
```
[the remark this answers is not in the document]
Comment (heyitsmaimai) [reply src/events/order/cd-event-sessions.vue]: as in 'There are some errors with your booking: there are no tickets left' or the other "error" that says 'We\'re sorry, there are no tickets left.'

[the remark this answers is not in the document]
Comment (Wardormeur) [reply src/events/order/cd-event-sessions.vue]: 2) is more human ? :)

[the remark this answers is not in the document]
Comment (heyitsmaimai) [reply src/events/order/cd-event-sessions.vue]: Hm yes..I am thinking what would be the most appropriate message for this error. bc they may think that all the tickets are gone?? something like 'we're sorry, one or more of your tickets is over capacity' or something?

[the remark this answers is not in the document]
Comment (Wardormeur) [reply src/events/order/cd-event-sessions.vue]: Surely works for me, particularly for an error that should never happen :p

[the remark this answers is not in the document]
Comment (heyitsmaimai) [reply src/events/order/cd-event-sessions.vue]: big facts

**Pull request merged**